docs/tutorial/index.rst

```diff
@@ -0,0 +1,270 @@
+and then click on the "+" button to the right of the name.
+The next page will ask you to fill some details about your Read the Docs project:
+
+Name
+  The name of the project. It has to be unique across all the service,
+  so it is better if you prepend your username,
+  for example ``astrojuanlu-rtd-tutorial``.
+
+Repository URL
+  The URL that contains the sources. Leave the automatically filled value.
+
+Repository type
+  Version control system used, leave it to "Git" (which should be the default).
+
+Default branch
+  Name of the default branch of the project, leave it to ``main``.
+
+Edit advanced project options
+  Leave it unchecked, we will make some changes later.
+
+After hitting the "Next" button, you will be redirected to the project home.
+You just created your first project on Read the Docs!
+
+.. figure:: /_static/images/tutorial/rtd-project-home.png
+   :width: 80%
+   :align: center
+   :alt: Project home
+
+   Project home
+
+Checking the first build
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs will try to build the documentation of your project
+right after you create it.
+To see the build logs,
+click on the "Your documentation is building" link on the dashboard,
+or alternatively navigate to the "Builds" page,
+then open the one on top (the most recent one).
+
+If the build has not finished yet by the time you open it,
+you will see a spinner next to a "Installing" or "Building" indicator,
+meaning that it is still in progress.
+
+.. figure:: /_static/images/tutorial/rtd-first-successful-build.png
+   :width: 80%
+   :align: center
+   :alt: First successful documentation build
+
+   First successful documentation build
+
+When the build finishes, you will see a green "Build completed" indicator,
+the completion date, the elapsed time,
+and a link to see the corresponding documentation.
+If you now click on "View docs", you will see your documentation live!
+
+.. figure:: /_static/images/tutorial/rtd-first-light.png
+   :width: 80%
+   :align: center
+   :alt: HTML documentation live on Read the Docs
+
+   HTML documentation live on Read the Docs
+
+Basic configuration changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can now proceed to make some basic configuration adjustments.
+For that, click on the "⚙ Admin" button, which will open the Settings page.
+
+First of all, add the following text in the description:
+
+    Lumache (/lu'make/) is a Python library for cooks and food lovers
+    that creates recipes mixing random ingredients.
+
+Then set the project homepage to ``https://world.openfoodfacts.org/``,
+and write ``food, python`` in the list of tags.
+All this information will be shown on your project home.
+
+After that, configure your email so you get a notification if the build fails.
+To do so, click on the "Notifications" link on the left,
+type the email where you would like to get the notification,
+and click the "Add" button.
+After that, your email will be shown under "Existing Notifications".
```

Comment:
The UX on this page is so bad :( Really need to improve it.

Reply:
Yep, it's ugly 😓